test/specs/buffer-while-by.coffee

```diff
@@ -20,12 +20,24 @@ describe 'bufferWhileBy', ->
       b = prop()
       expect(a.bufferWhileBy(b)).toActivate(a, b)
 
-    it 'should end when primary ends', ->
+    it 'should end when primary ends (w/o {emitEmpty: true})', ->
       expect(send(stream(), ['<end>']).bufferWhileBy(stream())).toEmit ['<end:current>']
       a = stream()
       b = stream()
       expect(a.bufferWhileBy(b)).toEmit ['<end>'], -> send(a, ['<end>'])
 
+    it 'should flush empty buffer and then end when primary ends (w/ {emitEmpty: true})', ->
+      expect(send(stream(), ['<end>']).bufferWhileBy(stream(), {emitEmpty: true})).toEmit [{current: []}, '<end:current>']
+      a = stream()
+      b = stream()
+      expect(a.bufferWhileBy(b, {emitEmpty: true})).toEmit [[], '<end>'], -> send(a, ['<end>'])
+
+    it 'should flush empty buffer and then end when secondary ends (w/ {flushOnChange: true, emitEmpty: true})', ->
```

Comment:
The test seems ok, but the title should be "should flush empty buffer when secondary emits false ...".